fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -405,12 +405,17 @@ class PythonUtils {
 		    «protectedArea(cls, attribute.name + "_write")»
 		    
     '''
-    def writeAttributeMethodHL(PogoDeviceClass cls, Attribute attribute) '''
-        def write_«attribute.name»(self, value):
+    def writeAttributeMethodHL(PogoDeviceClass cls, Attribute attribute, boolean isDynamic) '''
+        def write_«attribute.name»(self, «IF isDynamic»w_attr«ELSE»value«ENDIF»):
                 «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»
                 «openProtectedAreaHL(cls, attribute.name + "_write")»
                 """Set the «attribute.name» attribute."""
+                «IF isDynamic==false»
                 pass
+                «ELSE»
+                name = w_attr.get_name()
+                self.attr_«attribute.name»_read = w_attr.get_write_value()
```

Comment:
Since the `self.attr_«attribute.name»_read` is not initialised in the standard code generated by Pogo, suggest we comment this code out.  It would work, if uncommented, but since the read attribute method might not work, it seems better to have both read and write method commented out for consistency.
```suggestion
                """Example implementation:
                name = w_attr.get_name()
                self.attr_«attribute.name»_read = w_attr.get_write_value()
                """
```